### PR TITLE
Changed the border radius of search-bar to 8px

### PR DESCRIPTION
**Pull request by kaushiksriram16**

### CODE REVIEW

File: projects/commudle-admin/src/app/components/search-bar/search-bar.component.scss [hunk 1]
```diff
@@ -17,9 +17,13 @@
       input {
         height: 2.5rem;
         border: none;
-        border-radius: 0;
+        border-top-left-radius: 8px;
+        border-bottom-left-radius: 8px;
```
Comment (Ashwin-op) [inline projects/commudle-admin/src/app/components/search-bar/search-bar.component.scss]: Please use variables defined in variables.scss

File: projects/commudle-admin/src/app/components/search-bar/search-bar.component.scss [hunk 2]
```diff
@@ -43,12 +47,36 @@
         button {
           height: 100%;
           border: none;
-          border-radius: 0;
+          border-top-left-radius: 0;
+          border-bottom-left-radius: 0;
+          border-top-right-radius: 8px;
+          border-bottom-right-radius: 8px;
```
Comment (Ashwin-op) [inline projects/commudle-admin/src/app/components/search-bar/search-bar.component.scss]: Please use variables defined in variables.scss

File: projects/commudle-admin/src/app/components/search-bar/search-bar.component.scss [hunk 3]
```diff
@@ -57,12 +85,16 @@
     right: 0;
     z-index: 2;
     box-shadow: 0 7px 9px 0 rgba(100, 100, 111, 0.2);
-
+    border-bottom-left-radius: 8px;
```
Comment (Ashwin-op) [inline projects/commudle-admin/src/app/components/search-bar/search-bar.component.scss]: Please use variables defined in variables.scss

File: projects/commudle-admin/src/app/components/search-bar/search-bar.component.scss [hunk 4]
```diff
@@ -57,12 +85,16 @@
...
+    border-bottom-right-radius: 8px;
+    
     .search-results {
       height: 0;
       overflow: auto;
       background-color: $light-purple;
       transition: all 0.3s ease-in-out;
+      border-bottom-left-radius: 8px;
+      border-bottom-right-radius: 8px;
```
Comment (Ashwin-op) [inline projects/commudle-admin/src/app/components/search-bar/search-bar.component.scss]: Please use variables defined in variables.scss

File: projects/commudle-admin/src/app/components/search-bar/search-bar.component.html [hunk 5]
```diff
@@ -1,11 +1,11 @@
 <div (clickOutside)="toggleSearchSuffix(false, $event)" appClickOutside class="home-navbar-search">
-  <div class="d-flex search-container">
-    <nb-form-field (click)="toggleSearchSuffix(true)" class="search-form">
+  <div class="d-flex search-container" #searchContainer>
```
Comment (Ashwin-op) [inline projects/commudle-admin/src/app/components/search-bar/search-bar.component.html]: Why is this #searchContainer needed?

File: projects/commudle-admin/src/app/components/search-bar/search-bar.component.ts [hunk 6]
```diff
@@ -1,4 +1,4 @@
-import {Component, OnDestroy, OnInit} from '@angular/core';
+import {Component, ElementRef, OnDestroy, OnInit, ViewChild} from '@angular/core';
```
Comment (Ashwin-op) [inline projects/commudle-admin/src/app/components/search-bar/search-bar.component.ts]: Please remove unused imports

File: projects/commudle-admin/src/app/feature-modules/users/components/public-profile/user-basic-details/basic-user-profile/basic-user-profile.component.ts [hunk 7]
```diff
@@ -11,6 +11,8 @@ import {
   WhiteSpaceNotAllowedValidator
 } from 'projects/shared-helper-modules/custom-validators.validator';
 import { Router } from '@angular/router';
+import { NbDialogRef, NbDialogService } from '@nebular/theme';
```
Comment (Ashwin-op) [inline projects/commudle-admin/src/app/feature-modules/users/components/public-profile/user-basic-details/basic-user-profile/basic-user-profile.component.ts]: Please remove unused imports

File: projects/commudle-admin/src/app/feature-modules/users/components/public-profile/user-basic-details/basic-user-profile/basic-user-profile.component.ts [hunk 8]
```diff
@@ -126,11 +131,20 @@ export class BasicUserProfileComponent implements OnInit {
       if (data) {
         this.toastLogService.successDialog('Updated!');
```
Comment (Ashwin-op) [inline projects/commudle-admin/src/app/feature-modules/users/components/public-profile/user-basic-details/basic-user-profile/basic-user-profile.component.ts]: Let's comment this toaster notification

**Review by Ashwin-op — CHANGES_REQUESTED** [review]

File: projects/commudle-admin/src/app/components/search-bar/search-bar.component.html (hunk 5 above)
Comment (kaushiksriram16) [reply projects/commudle-admin/src/app/components/search-bar/search-bar.component.html]: It was from the previous commit when I used ViewChild 😅

**Review by Ashwin-op — APPROVED** [review]

**Comment by arpansac** [comment]
Good Job @kaushiksriram16!!

Thanks for the review @Ashwin-op :)

**Pull request merged**